tests/update/patch_before_migration.pm

```diff
@@ -58,8 +58,9 @@ sub patching_sle {
         # Update origin system on zVM that is controlled by autoyast profile and reboot is done by end of autoyast installation
         # So we skip reboot here after fully patched on zVM to reduce times of reconnection to s390x
         if (!get_var('UPGRADE_ON_ZVM')) {
+            assert_script_run 'sync';
```

Comment:
You need add longer timeout here, since default timeout 30s seems not enough for disk syncing in case worker is heavy loaded.

Reply:
Qingming, as we talked offline I updated the code to just do sync with timeout of 600s.